modules/ve-cx/init/ve.init.mw.CXTarget.js

```
@@ -1103,11 +1113,21 @@ ve.init.mw.CXTarget.prototype.changeContentSource = function (
  *
  * @param {number} sectionNumber
  */
-ve.init.mw.CXTarget.prototype.prefetchTranslationForSection = function ( sectionNumber ) {
+ve.init.mw.CXTarget.prototype.prefetchTranslationForSection = function ( sectionNumber, set_it ) {
```

ESLint

[error] Identifier 'set_it' is not in camel case.

(camelcase)